### PR TITLE
Affine Cipher Cryptography using Java

### PR DESCRIPTION
**Pull request by PrasadMulherkar**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given the input to the Affine Cipher algorithm, it produces the coded message for the given input, and vise versa, that is given encrypted message to the Affine Cipher algorithm, it produces the original message.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#486 Affine Substitution Cipher

### CODE REVIEW

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@PrasadMulherkar Please add the **README** file details & fill the PR template by mentioning the issue number properly

File: Java/cryptography/AffineCipher.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+ * 			4) b => It is a addition factor. There are no specific constraints other than 0 <= b <= z.
+ */
+
+import java.util.*;
+
+public class AffineCipher {
+	//Creating objects that will be used throughout the code. 
+	List<Character> alphabets = new ArrayList<Character>();
+	Random randomObject = new Random();
+	int z, a, b, a_inverse;
```
Comment (atarax665) [inline Java/cryptography/AffineCipher.java]: Use meaningful variable names.

File: Java/cryptography/AffineCipher.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+			}
+			
+			System.out.println("Do you want to continue (Y/N) ? ");
+			string = scanner.next();
+			
+		}while(string.equals("Y"));
+		
+		scanner.close();
+	}
+}
```
Comment (atarax665) [inline Java/cryptography/AffineCipher.java:157]: Add sample input/output in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by PrasadMulherkar** [comment]
I have changed according to the suggestions, please review them.

**Review by siddharth25pandey — APPROVED** [review]
@PrasadMulherkar Good to go
Able to merge
Thanks!

File: Java/cryptography/AffineCipher.java [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+ * 
+ * 			3) Inverse of Multiplication Factor => 	
+ * 					This is the a-inverse i.e. the multiplicative inverse of a with respect to length of alphabets. 
+ * 					Means (Multiplication Factor x Inverse Multiplication Factor) mod length of alphabets = 1.
+ * 
+ * 			4) Addition Factor => 
+ * 					It is a addition factor. There are no specific constraints other than 0 <= Addition Factor <= Length of alphabets.
+ */
+
+package hakinCodes.affineCipher;
```
Comment (vatsalkesarwani12) [inline Java/cryptography/AffineCipher.java]: remove package

File: Java/cryptography/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+AFFINE CIPHER :-
```
Comment (vatsalkesarwani12) [inline Java/cryptography/README.md]: delete this file

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
update the readme file and do all the changes
ASAP.

**Review by iamrajiv — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
@PrasadMulherkar just address the @plazzy99 comments, rest LGTM :+1:

**Comment by PrasadMulherkar** [comment]
Sorry for being late... I have made changes suggested by @plazzy99, hope this PR will be okay now

**Review by vatsalkesarwani12 — COMMENTED** [review]
update readme file

**Comment by PrasadMulherkar** [comment]
You told me to delete the readme file now which file to update??

**Comment by vatsalkesarwani12** [comment]
> You told me to delete the readme file now which file to update??

update the existing readme file of java
You can refer to any merged pr for reference

**Comment by PrasadMulherkar** [comment]
Hey I changed it please review it and tell me...
As I am new at it I tried my best please help me if I am wrong

**Comment by HarshCasper** [comment]
I guess we can solve it out while cleaning out the Repository. Thanks for the PR @PrasadMulherkar 

Merging it!

**Comment by PrasadMulherkar** [comment]
Thanks😀

**Pull request merged**